python/eo_acquisition.py

```diff
@@ -416,6 +423,195 @@ def compute_exptime(self, target_counts, meas_flux):
         exptime = min(max(exptime, self.exptime_min), self.exptime_max)
         return exptime
 
+    def getParallelHighConfig(self):
+        """
+        get phi0, phi1, phi2 existing values of pclkHighP
+        """
+        command = "printConfigurableParameters"
+        res = str(self.sub.ts8dac0.synchCommand(10, command).getResult())
+        m = re.search(r"pclkHighP: ([-\d]+\.\d+),", res)
+        if not m:
+            self.logger.info("m is None, res={}".format(res))
```

Comment:
The preferred way to format logging messages is like this:
```
self.logger.info("m is None, res=%s", res)
```
This is so the message formatting can be deferred until it is needed.  This isn't a big deal, but we should probably get in the habit of using the recommended practice.

Reply:
Sure.  That looks a lot closer to sprintf() style which is what python should have used.